app/_src/gateway/kong-enterprise/secrets-management/advanced-usage.md

```diff
@@ -128,8 +128,8 @@ Most of the vaults also support secret rotation by using TTLs:
 
 Parameter | Field name | Description
 ----------|------------|------------
-`vaults.config.ttl` | **TTL** | Time-to-live (in seconds) of a secret from the vault when cached by this node.
```

Comment:
Do we need to mention the default values in seconds for `ttl`s?

Reply:
It has no default value but it is constrained to between 0 seconds and 3 years.